Maven-git-practice/src/main/java/LinkedList.java

```diff
@@ -1,32 +1,16 @@
 public class LinkedList {
-    private Node head;
+
     private long size;
     public LinkedList(){
-        this.size=0L;
+        this.size=0;
     }
-    public void addNode(Object nodeData){
-        Node node=new Node();
-        node.data=nodeData;
-        node.next=null;
-        if(this.head==null){
-            this.head=node;
-            size++;
-        }else{
-            Node n=head;
-            //looping over the nodes until we find the last one
-            while(n.next!=null) {
-                n=n.next;
-            }
-            n.next=node;
-            size++;
-        }
-
+    public void addNode(int node){
+        size++;
     }
     public long size(){
         return this.size;
     }
+
 }
 class Node{
-    public Object data;
```

Comment:
If you to make node and linked list generic better use a generic class 
`class Node<T>{
         public T data;`